linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/SWasmGen.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.backend.wasmemitter
+
+import org.scalajs.ir.Names._
+import org.scalajs.ir.Trees.JSNativeLoadSpec
+import org.scalajs.ir.Types._
+
+import org.scalajs.linker.backend.webassembly._
+import org.scalajs.linker.backend.webassembly.Instructions._
+
+import VarGen._
+
+/** Scala.js-specific Wasm generators that are used across the board. */
+object SWasmGen {
+
+  def genBooleanNot(fb: FunctionBuilder): Unit = {
+    fb += I32Const(1)
+    fb += I32Xor
```

Comment:
Can't we use `I32Eqz` for this? IIUC the CoreWasmLib does that already. (means its probably not worth having the helper?)